chart/values.yaml

```diff
@@ -3597,19 +3597,23 @@ internalRedis:
     persistence:
       enabled: false
 
-## @section External Redis&trade; parameters
-## External Redis&trade; configuration
+## @section External Redis parameters
+## External Redis configuration
 ## All of these values are only used when redis.enabled is set to false
-## @param externalRedis.host Redis&trade; host
-## @param externalRedis.port Redis&trade; port number
-## @param externalRedis.password Redis&trade; password
-## @param externalRedis.existingSecret Name of an existing secret resource containing the Redis&trade; password in a 'redis-password' key
+## @param externalRedis.host Redis host
+## @param externalRedis.port Redis port number
+## @param externalRedis.password Redis password
+## @param externalRedis.tls_enabled Whether or not connect to Redis via TLS
+## @param externalRedis.tls_ca CA certificate in case Redis TLS cert it's selfsigned
+## @param externalRedis.existingSecret Name of an existing secret resource containing the Redis password in a 'redis-password' key
 ## @param externalRedis.existingSecretPasswordKey Key of the existing secret
 ##
 externalRedis:
   host: localhost
   port: 6379
   password: ""
+  tls_enabled: false
+  tls_ca: ""
```

Comment:
why are we using `_` here? all other variables are with other format

Reply:
true, I'm changing it 👍